ecommerce/hubspot_api_test.py

```diff
@@ -0,0 +1,122 @@
+"""
+Hubspot API tests
+"""
+# pylint: disable=redefined-outer-name
+from unittest.mock import Mock
+
+import pytest
+import factory
+from django.conf import settings
+from rest_framework import status
+
+from ecommerce.hubspot_api import (
+    send_hubspot_request,
+    HUBSPOT_API_BASE_URL,
+    make_sync_message,
+    make_contact_sync_message,
+)
+from users.serializers import UserSerializer
+
+
+@pytest.fixture()
+def hubspot_get_request(mocker):
+    """Mock a 200 response from Hubspot for successful user update"""
+    yield mocker.patch(
+        "ecommerce.hubspot_api.requests.get", return_value=Mock(status_code=status.HTTP_200_OK)
+    )
+
+
+@pytest.fixture()
+def hubspot_post_request(mocker):
+    """Mock a 200 response from Hubspot for successful user update"""
+    yield mocker.patch(
+        "ecommerce.hubspot_api.requests.post", return_value=Mock(status_code=status.HTTP_200_OK)
+    )
+
+
+@pytest.fixture()
+def hubspot_put_request(mocker):
+    """Mock a 200 response from Hubspot for successful user update"""
+    yield mocker.patch(
+        "ecommerce.hubspot_api.requests.put", return_value=Mock(status_code=status.HTTP_200_OK)
+    )
+
```

Comment:
I parametrized it on the method instead to consolidate the code. Also, since those fixtures were each being called only once and the return argument didn't matter, I defined cut them and created the patches inline.